prow/cmd/horologium/main.go

```diff
@@ -101,24 +101,24 @@ func main() {
 	}
 	cluster, err := cluster.New(cfg, func(o *cluster.Options) { o.Namespace = configAgent.Config().ProwJobNamespace })
 	if err != nil {
-		logrus.WithError(err).Fatal("failed to construct prowjob client")
+		logrus.WithError(err).Fatal("Failed to construct prowjob client")
 	}
 	// Trigger cache creation for ProwJobs so the following cacheSync actually does something. If we don't
 	// do this here, the first List request for ProwJobs will transiently trigger cache creation and sync,
 	// which doesn't allow us to fail the binary if it doesn't work.
 	if _, err := cluster.GetCache().GetInformer(interrupts.Context(), &prowapi.ProwJob{}); err != nil {
-		logrus.WithError(err).Fatal("failed to get a prowjob informer")
+		logrus.WithError(err).Fatal("Failed to get a prowjob informer")
 	}
 	interrupts.Run(func(ctx context.Context) {
 		if err := cluster.Start(ctx); err != nil {
-			logrus.WithError(err).Fatal("Cache failed to start")
+			logrus.WithError(err).Fatal("Cluster failed to start")
 		}
-		logrus.Info("Cache finished gracefully.")
+		logrus.Info("Cluster finished gracefully.")
```

Comment:
I think the original text was more correct. This now reads as if we were spinning up a new cluster, but we're actually creating a controller-runtime manager (informer?) to monitor the cluster.

Reply:
Okay, but we can consider using controller/manager